packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -271,15 +271,23 @@ export class ClusterReachability extends EventsScope {
    * @param {string} protocol
    * @param {number} latency
    * @param {string|null} [publicIp]
+   * @param {number|null} [serverPort]
    * @returns {void}
    */
-  private saveResult(protocol: 'udp' | 'tcp' | 'xtls', latency: number, publicIp?: string | null) {
+  private saveResult(
+    protocol: 'udp' | 'tcp' | 'xtls',
+    latency: number,
+    publicIp?: string | null,
+    serverPort?: number | null
+  ) {
     const result = this.result[protocol];
 
     if (result.latencyInMilliseconds === undefined) {
+      const port = serverPort ? `(${serverPort})` : '';
+
       LoggerProxy.logger.log(
         // @ts-ignore
-        `Reachability:index#saveResult --> Successfully reached ${this.name} over ${protocol}: ${latency}ms`
+        `Reachability:index#saveResult --> Successfully reached ${this.name} over ${protocol}${port}: ${latency}ms`
```

Comment:
we only store and log the first result, so this log can be a bit misleading - the log saying that we reached some cluster over UDP(5004) doesn't mean that we failed to reach it on port 9000, it just means that we got the ice candidate from 5004 quicker than 9000

Reply:
Do you think it's about rephrasing, or we generally don't need this info?